src/Executor/Values.php

```diff
@@ -94,7 +94,7 @@ public static function getVariableValues(Schema $schema, $definitionNodes, array
      * @return array
      * @throws Error
      */
-    public static function getArgumentValues($def, $node, $variableValues)
+    public static function getArgumentValues($def, $node, $variableValues = null)
```

Comment:
It is a relaxed requirement, all existing calls to this method will work as they do now. I don't see how it is a BC break?